### DISCUSSION TITLE
GitHub Actions secrets CLI tool

### DISCUSSION THREAD

**Discussion by unfor19** [community/community#26517] [Actions]
<p>Hi, I’ve created a simple CLI that can help you manage your GitHub Actions secrets</p>
<p><strong>githubsecrets</strong></p>
<p>Official page - <a href="https://github.com/unfor19/githubsecrets" title="githubsecrets" rel="nofollow noopener">https://github.com/unfor19/githubsecrets</a></p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/d/dd990e3a5a4abc595c17dd585bb01ce50445df49.gif" data-download-href="/uploads/short-url/vClrlkoJEQEGbHTVBEmJxA6hrw5.gif?dl=1" title="githubsecrets-demo.gif" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/15122452/181098124-eef2cb08-115a-4c35-88f2-5d3b2b0f6750.gif" alt="githubsecrets-demo.gif" data-base62-sha1="vClrlkoJEQEGbHTVBEmJxA6hrw5" width="690" height="273" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/d/dd990e3a5a4abc595c17dd585bb01ce50445df49_2_690x273.gif, https://aws1.discourse-cdn.com/github/optimized/2X/d/dd990e3a5a4abc595c17dd585bb01ce50445df49_2_1035x409.gif 1.5x, https://aws1.discourse-cdn.com/github/original/2X/d/dd990e3a5a4abc595c17dd585bb01ce50445df49.gif 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/d/dd990e3a5a4abc595c17dd585bb01ce50445df49_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">githubsecrets-demo.gif</span><span class="informations">1072×425 315 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>
<p><strong>Install with pip</strong></p>
<pre><code>$ pip install githubsecrets
...
$ ghs --help
...
</code></pre>
<p><strong>Docker</strong></p>
<p>Or use with Docker; you must supply all arguments, prompts are not available in Docker mode</p>
<p><strong>macOS and Linux</strong></p>
<pre><code>$ docker run -v $HOME:/root unfor19/githubsecrets --help
Usage: ghs [OPTIONS] COMMAND [ARGS]...
</code></pre>
<p><strong>Windows</strong></p>
<pre><code>$ docker run --rm -v c:/Temp:/root unfor19/githubsecrets --help
Usage: ghs [OPTIONS] COMMAND [ARGS]...</code></pre>

**Comment by Yanjingzhu** [comment, marked as answer]
<p>Hi unfor19, </p>
<p>Wow, what a wonderful tool. I am sure It is helpful for other communities who wants to use secrets. Thank you for your contribution to GitHub Actions. </p>

**Comment by unfor19** [comment]
<p>Thank you Yanjingzhu, much appreciated!</p>
<p>I’m marking your answer as ‘an accepted solution’ since my post is not a real question</p>
